### DISCUSSION TITLE
How to display text as subtitles in designated positions on the screen ?

### DISCUSSION THREAD

**Discussion by webclient163** [SuRGeoNix/Flyleaf#477] [Q&A]
I am using it to play real-time H264 videos and hope to display the values collected by sensors on the player. Is there a simple way to make the player display real-time text?

**Comment by SuRGeoNix** [comment]
Ideally, this should be coming from the source side as a data stream which will automatically set the timestamp same as last video timestamp. You might be able to pass this also with srt/ass format and open it as external subtitles stream. The issue with this is the timestamps need to much the video to be able to sync it.

**Reply by webclient163** [reply]
Thank you! I am considering adding a mask layer to the video to display the content I need to display
